swagger.yaml

```diff
@@ -6680,6 +6683,7 @@ definitions:
       - 'apps-domain-name-resolved-correctly'
       - 'compatible-with-cluster-platform'
       - 'dns-wildcard-not-configured'
+      - 'has-min-requirements-for-disk-encryption'
```

Comment:
For consistency

```suggestion
      - 'disk-encryption-requirements-satisfied'
```

Reply:
Most of the HW validations ID starts with `has-min...`.
Consistency with what? seems like `has-min` is more consistent from what I can see.
Also https://github.com/openshift/assisted-service/pull/2482#discussion_r696706339

Reply:
It just seems too long and wordy.  I was looking at:
      - 'lso-requirements-satisfied'
      - 'ocs-requirements-satisfied'

I think it's a better format